nixos/modules/tasks/network-interfaces.nix

```diff
@@ -1246,7 +1246,7 @@ in
             '';
 
             # Udev attributes for systemd to name the device and to create a .device target.
-            systemdAttrs = n: ''NAME:="${n}", ENV{INTERFACE}:="${n}", ENV{SYSTEMD_ALIAS}:="/sys/subsystem/net/devices/${n}", TAG+="systemd"'';
+            systemdAttrs = n: ''NAME:="${n}", ENV{INTERFACE}="${n}", ENV{SYSTEMD_ALIAS}="/sys/subsystem/net/devices/${n}", TAG+="systemd"'';
```

Comment:
looks like maybe you didn't fix the first one?

Reply:
Oh, that's the NAME key, maybe it's different? Where is the documentation for this?

Reply:
Yes, only the `ENV` assignments need fixing. Unless I'm missing something in [udev's manual](https://www.freedesktop.org/software/systemd/man/udev.html) it looks like `ENV{FOO}:=` could be valid, but it is not.
I tried to search for `"ENV key takes" udev`, to no avail.
I searched examples, they all use `ENV{FOO}=`, not `ENV{FOO}:=`, eg. https://github.com/M0Rf30/android-udev-rules/blob/master/51-android.rules

Reply:
https://github.com/systemd/systemd/blob/cfb6197bc31eb6b2631dec7bf8d7a253e7891016/src/udev/udev-rules.c#L616 found the line where final assignment is disallowed for env!